test/functional/framework.go

```diff
@@ -462,7 +462,7 @@ func (f *FluentdFunctionalFramework) WritesNApplicationLogsOfSize(numOfLogs, siz
 func (f *FluentdFunctionalFramework) WriteMessagesToLog(msg string, numOfLogs int, filename string) error {
 	logPath := filepath.Dir(filename)
 	encoded := base64.StdEncoding.EncodeToString([]byte(msg))
-	cmd := fmt.Sprintf("mkdir -p %s;for n in {1..%d};do echo \"$(echo %s|base64 -d)\" >> %s;sleep 1s;done", logPath, numOfLogs, encoded, filename)
+	cmd := fmt.Sprintf("mkdir -p %s;for n in {1..%d};do echo \"$(echo %s|base64 -d)\" >> %s;done", logPath, numOfLogs, encoded, filename)
```

Comment:
@jcantrill This sleep is a bandaid for a flake where fluent doesn't "notice" this log.

Reply:
Did removal of this sleep fix that test?

Reply:
reverted